io.github.Fndroid.clash_for_windows.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.Fndroid.clash_for_windows
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+base: org.electronjs.Electron2.BaseApp
+base-version: 21.08
+sdk: org.freedesktop.Sdk
+command: cfw.sh
+
+finish-args:
+- --share=ipc
+- --share=network
+- --socket=x11
+- --filesystem=xdg-config
```

Comment:
In addition to .config/clash, there seems to be a .config/clash_win, and the functions that start with Linux need to write files to the autostart folder, so here we write the xdg-confg directory directly for convenience.

Reply:
There are `config/clash_win` and `config/clash`.  The first is for electron data, another is for clash linux.  
`clash_win` is in `~/.var/app/io.github.Fndroid.clash_for_windows/config`  
`clash` is in `~/.config/clash`  
Only need add permission for `clash`, using persist to make it in sandbox  
Using `xdg-config/autostart:create` for minimal permission, rather than entire `xdg-config`